app/globals.css

```diff
@@ -115,6 +115,145 @@
   --sidebar-ring: oklch(0.552 0.016 285.938);
 }
 
+.dark body {
+  background: linear-gradient(180deg, #0f1419, #1a1f2e);
+}
+
+:root:not(.dark) body {
+  background: #fafafa;
+}
+
+.result-container {
```

Comment:
since there is only one instance for this className instead directly use tailwind css classes